scripts/zypp-plugin.py

```diff
@@ -56,6 +56,9 @@ class Config:
 
     def __init__(self):
         self.solvables = []
+	self.zypper_extended_description = []
+	self.zypper_extended_description.append("false")
+	self.zypper_extended_description.append("0")
```

Comment:
Don't use an array to store two values of different type and meaning. Just use e.g. ```self.zypper_extended_description_enable = false``` and ```self.zypper_extended_description_length = 0```.